backend/prompt_studio/prompt_studio_core/prompt_studio_helper.py

```diff
@@ -795,6 +795,7 @@ def dynamic_indexer(
         try:
 
             usage_kwargs = {"run_id": run_id}
+            usage_kwargs["file_name"] = filename
```

Comment:
@johnyrahul during processing since we copy and alter the file name / path to `INFILE` .etc. - shall we explicitly call this `source_file_name` or `original_file_name` to distinguish between the actual name and something we changed?
Its a bit confusing since we set these for tracking usage and we also pass the `file_path` during indexing

Reply:
Added comments to avoid confusion